### ISSUE TITLE
Publish UA-I Personal Terms of Service

### ISSUE THREAD

**Issue by matthewpaulthomas** [canonical-web-and-design/ubuntu.com#5931]
[Source document](https://docs.google.com/document/d/1b2TSeqGfBsmZMepXGEGkEWyTDo0M5aHmyqaQIzEZfPs/edit).

In #5930 I have assumed that the URL will be `/legal/ubuntu-advantage/personal`.

This page should also be linked from a new card on [the UA legal page](https://ubuntu.com/legal/ubuntu-advantage).

**Comment by matthewpaulthomas** [comment]
Currently blocked on finalising the terms.

**Comment by matthewpaulthomas** [comment]
Terms are now ready.
